src/containers/SignUp/Form/Form.tsx

```diff
@@ -9,26 +9,61 @@ import * as S from "./emotion";
 import { useState } from "react";
 
 export default function Form() {
+  const [isAvailableId, setIsAvailableId] = useState(false);
+  const [userInfo, setUserInfo] = useState({
+    loginType: "email",
+    name: "",
+    email: "",
+    role: "admin",
+    password: "",
+    phoneNumber: "",
+    grade: 0,
+    gender: "",
+  });
+  const [maleChecked, setMaleChecked] = useState(false);
+  const [femaleChecked, setFemaleChecked] = useState(false);
```

Comment:
이건 그냥 하나의 상태로 
```suggestion
  const [gender, setGender] = useState<"male" | "female">("male"); 
```
이렇게 관리하는게 나을듯

Reply:
근데 애초에 이게 필요한 상태인지 잘 모르겠는데
위에서 userInfo에서 어차피 다 관리되고있는데 굳이 상태를 더 나눠야하나요?

Reply:
중복 선택을 방지하려고 input checkbox 의 checked 속성 관리하려고 만든 상태입니다.
두개 선언해서 쓰는거 좀 보기 그렇긴한데 좋은 방법 있을까요

Reply:
```tsx
  <input
            type="checkbox"
            value="male"
            id="male"
            checked={userInfo.gender === "male"}
   />
```
이렇게 하면 되지 않나